src/main/java/org/embulk/input/zendesk/ZendeskInputPlugin.java

```diff
@@ -453,4 +480,35 @@ private void validateUserEvent(PluginTask task)
             }
         }
     }
+
+    private void checkValidTimeRangeThrowException(PluginTask task)
```

Comment:
Why don't we validate this method content in `validateTime` if it is in preview mode? `guess` is also `preview` mode.

Reply:
fixed